src/stylesheets/components/_process-list.scss

```diff
@@ -0,0 +1,62 @@
+// PROCESS LIST
+$bullet-border-width: units($theme-process-list-connection-line-width) / 2;
+$padding-left: ((units($theme-process-list-bullet-size) / 2) - (units($theme-process-list-connection-line-width) / 2));
+
+.usa-process-list {
+    @include typeset($theme-process-list-bullet-font-family);
+    @include unstyled-list;
+    counter-reset: numbered-list;
+    padding: 20px 0 0 $padding-left;
+    position: relative;
+}
+
+.usa-process-list__item {
+    border-left: units($theme-process-list-connection-line-width) solid color($theme-process-list-connection-line-color);
+    font-size: 16px;
+    padding-bottom: units(4);
+    padding-left: units($theme-process-list-bullet-size);
+    &:last-child {
+        border-left: none;
+    }
+    &::before {
+        @include u-flex("row", "align-center", "justify-center");
+        $half-of-font-size: -0.5em;
+        background-color: color($theme-process-list-bullet-color);
+        border-radius: 50%;
+        border: $bullet-border-width solid color($theme-process-list-bullet-stroke-color);
+        color: color($theme-process-list-bullet-text-color);
+        content: counter(numbered-list, decimal);
+        counter-increment: numbered-list;
+        display: flex;
+        font-family: family($theme-process-list-bullet-font-family);
+        font-size: size($theme-process-list-bullet-font-family, $theme-process-list-heading-font-size);
+        font-weight: font-weight("bold");
+        height: units($theme-process-list-bullet-size);
+        left: 0;
+        line-height: 1.2;
+        margin-top: $half-of-font-size;
+        position: absolute;
+        width: units($theme-process-list-bullet-size);
+    }
+}
+
+.usa-prose {
+    > .usa-process-list {
```

Comment:
Do you know if the child selector is sufficient here? What happens if you were to place this component inside of a `div` inside of `.usa-prose`. Any breakage if you don't use the `>`?

Reply:
Since I think this is compensating for `usa-prose` styling and `usa-prose` styling is scoped to direct children, I think this is correct